src/libs/ReportActionsUtils.ts

```diff
@@ -560,6 +564,23 @@ function shouldHideNewMarker(reportAction: OnyxEntry<ReportAction>): boolean {
     return !isNetworkOffline && reportAction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
 }
 
+/**
+ * Checks whether an action is actionable track expense.
+ *
+ */
+function isActionableTrackExpense(reportAction: OnyxEntry<ReportAction>): boolean {
```

Comment:
```suggestion
function isActionableTrackExpense(reportAction: OnyxEntry<ReportAction>): reportAction is ReportActionBase & OriginalMessageActionableTrackedExpenseWhisper {
```

We should retain the type predicate so that TS knows the `originalMessage` is track whisper when this returns `true`.